packages/core/src/events.ts

```diff
@@ -51,6 +58,7 @@ export interface EventPaginationOptions extends PaginationOptions {
   transactionHash?: string;
   status?: EventStatusType;
   blockchain?: BlockchainType;
+  symbol?: string;
```

Comment:
`symbol` should not be here. It should be in `ValueTransferEventPagniationOptions`

Reply:
Thank you for your comment! I'll fix and push it.